ios/Podfile

```diff
@@ -21,21 +49,30 @@ target 'HathorMobile' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
+  #
+  # OBS1: Disabled by default as this conflicts with use_frameworks!,
+  # which is required by Firebase. Besides that, it is used only for development.
+  #
+  # OBS2: There is a fix for this in RN 0.69.1
+  #
+  # use_flipper!()
```

Comment:
We are currently not using flipper anyways, right?

Chrome dev tools will continue working as usual

Reply:
Yes, we are still able to use the debugger-ui from the browser.